src/bundle/Resources/views/themes/admin/ui/component/input_date_and_time.html.twig

```diff
@@ -0,0 +1,36 @@
+{% import '@ibexadesign/ui/component/macros.html.twig' as html %}
```

Comment:
I think it may be better to put this file inside its own directory:
`component/inputs/input_date_and_time.html.twig` 
WDYT?

Reply:
But then we should probably move `input_text.html.twig` widget as well, WDYT guys? @lucasOsti @dew326